build/rpms/cri-tools.spec

```diff
@@ -0,0 +1,21 @@
+Name: cri-tools
+Version: OVERRIDE_THIS
+Release: 00
+License: ASL 2.0
+Summary: Container Runtime Interface tools
+
+URL: https://kubernetes.io
+
+%description
+Binaries to interface with the container runtime.
+
+%prep
+# TODO(chuckha): update this to use %{version} when the dash is removed from the release
+tar -xzf {crictl-v1.0.0-beta.1-linux-amd64.tar.gz}
```

Comment:
You could set a shell variable before the tar command to replace the '_' with a '-' to avoid duplicating the version here.

Reply:
Let's track it in a ticket as discussed offline and leave this TODO here for now.

Reply:
https://github.com/kubernetes/kubeadm/issues/903